### PR TITLE
Dynamic response for editing user info & seller product error's handled

### PR DESCRIPTION
**Pull request by Aykchen**

### CODE REVIEW

File: ecommerce-website/src/SystemVariables.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
-const baseUrl = "https://heagle-backend.herokuapp.com";
-// const baseUrl = "http://localhost:3001"
+// const baseUrl = "https://heagle-backend.herokuapp.com";
+const baseUrl = "http://localhost:3001"
```
Comment (Its-Alann) [inline ecommerce-website/src/SystemVariables.js:2]: Make sure to comment line 2 and uncomment line 1

File: server/server.js [hunk 2]
```diff
@@ -8,8 +8,8 @@ app.use(cors());
 app.use(express.json());
 
 app.use(function (req, res, next) {
-	res.header("Access-Control-Allow-Origin", "https://heagle.herokuapp.com"); // update to match the domain you will make the request from
-	// res.header("Access-Control-Allow-Origin", "http://localhost:3000"); // update to match the domain you will make the request from
+	// res.header("Access-Control-Allow-Origin", "https://heagle.herokuapp.com"); // update to match the domain you will make the request from
+	res.header("Access-Control-Allow-Origin", "http://localhost:3000"); // update to match the domain you will make the request from
```
Comment (Its-Alann) [inline server/server.js:12]: Line comments

**Review by georgia1515 — APPROVED** [review]
Looks good!

**Review by Its-Alann — CHANGES_REQUESTED** [review]
Check the comments for local host

File: ecommerce-website/src/SystemVariables.js (hunk 1 above)
Comment (Aykchen) [reply ecommerce-website/src/SystemVariables.js]: Yes, you're right! Updated!

File: server/server.js (hunk 2 above)
Comment (Aykchen) [reply server/server.js]: Yes, you're right! Updated too!

**Review by Its-Alann — APPROVED** [review]
Looks good

**Pull request merged**